### ISSUE TITLE
[Feature Request]  Edited the wiki page to add a new section about MacOS initial setup.

### ISSUE THREAD

**Issue by Colin6618** [Ryujinx/Ryujinx#5084]
### Overview

Considering that Ryujinx already supports MacOS, but there is no installation guide on the wiki page.

*Changes*
I edited the wiki page, (in my fork repo, I cannot edit Ryujinx wiki), to add a new section about MacOS initial setup.

*Link*
https://github.com/Colin6618/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide#initial-setup---placement-of-prodkeys

### Smaller details

NA

### Nature of request

This is not a new feature, nor is it a bug, sorry for choosing the new feature tag.

### Why would this feature be useful?

The MacOS application directory is not exactly the same as the Windows directory, so users will need the inital setep instruction.

**Comment by AcK77** [comment]
Isn't the configuration exact the same thing as all other OS? Because based on what you edited it's the same.

**Comment by Colin6618** [comment]
> Isn't the configuration exact the same thing as all other OS? Because based on what you edited it's the same.

Yes, the installation process is exactly the same, also copied to the system folder. The only difference is that the path and the contents of of the macOS ryujinx folder are not exactly the same as those of Windows, which may cause some users to be confused. That's why I added some screenshots to help.

**Comment by AcK77** [comment]
Then the **5.** should just handle specific OS path. Other screenshots aren't really needed. But I agree we should precise it a bit more.

**Comment by gdkchan** [comment]
That said, we'll need to update the guide with Avalonia screenshots eventually.

**Comment by ColinHanCertik** [comment]
> Then the **5.** should just handle specific OS path. Other screenshots aren't really needed. But I agree we should precise it a bit more.

I agree, it's better this way, and wiki doesn't become too verbose.

**Comment by Colin6618** [comment]
> Then the **5.** should just handle specific OS path. Other screenshots aren't really needed. But I agree we should precise it a bit more.

I agree, it's better this way, and wiki doesn't become too verbose.
